mobile/lib/modules/login/ui/login_form.dart

```diff
@@ -317,6 +326,10 @@ class LoginForm extends HookConsumerWidget {
                       ],
                     ],
                   ),
+            if (!isOauthEnable.value && !isPasswordLoginEnable.value)
+              const Center(
+                child: Text("Login has been disabled."),
```

Comment:
How to support translation? Just add a field to the English json?

Reply:
Correct, and the end of the Text it should have a `tr()` method. You can reference other files for example